pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md

```diff
@@ -13,12 +13,70 @@ Notre Load Balancer Public Cloud est basé sur le service [Openstack Octavia](ht
 ## Prérequis
 
 - Un [projet Public Cloud](https://www.ovhcloud.com/fr/public-cloud/)
-- Utiliser [l'environnement de commande Openstack](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api)
-- Avoir installé le client Openstack Octavia
+- Comprendre les [concepts du Load Balancer](/pages/public_cloud/public_cloud_network_services/concepts-03-loadbalancer)
+- Comprendre les [concepts de réseau Public Cloud](/pages/public_cloud/public_cloud_network_services/concepts-01-public-cloud-networking-concepts)
+- Un Load Balancer nécessite un sous-réseau, lisez [ce guide](/pages/public_cloud/public_cloud_network_services/getting-started-07-creating-vrack) pour plus d'informations
+- (Facultatif) : Ce guide explique la configuration du load balancer via l'interface graphique et l'interface en ligne de commande. Si vous souhaitez utiliser cette dernière, installez [l'environnement en ligne de commande OpenStack](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api)
+
 
 ## En pratique
 
-### Configurer votre réseau privé
+### Création du Load Balancer depuis votre espace client OVHcloud
+
+Ouvrez votre projet Public Cloud et cliquez sur `Load Balancer`{.action} (sous `Network`) dans le menu de gauche, puis cliquez sur le bouton `Créer un Load Balancer`{.action}.
+
+La page de configuration s'ouvre.
+
+#### Étape 1 : Choix de la taille
+
+![Choix de la taille](images/size.png){.thumbnail}
+
+L'interface contient un lien vers notre site sur lequel les caractéristiques / benchmark de toutes tailles sont fournies. Une fois que vous avez choisi votre taille, cliquez sur `Suivant`{.action}.
+
+#### Étape 2 : Choix de la région
+
+![Choix de la région](images/region.png){.thumbnail}
+
+Seules les régions sur lesquelles vous avez un réseau privé et au moins un sous-réseau peuvent être sélectionnées. Sélectionnez la région et cliquez sur `Suivant`{.action}.
+
+#### Étape 3 : attacher une IP publique (ou non)
+
+![Public IP choice](images/IP_flottante.png){.thumbnail}
```

Comment:
image name seems weird

Reply:
Oops, this shouldn't have been translated. I'll fix this.